### PR TITLE
Initial dracut overlay files

### PR DESCRIPTION
**Pull request by smarlowucf**

### CODE REVIEW

File: etc/dracut.conf.d/07-aws-type-switch.conf [hunk 1]
```diff
@@ -0,0 +1 @@
+add_drivers+=" ena xen-blkfront xen-netfront "
```
Comment (apozsuse) [inline etc/dracut.conf.d/07-aws-type-switch.conf]: should this `aws` file be included in google's config? :thinking:

Comment (rjschwei) [reply etc/dracut.conf.d/07-aws-type-switch.conf]: This is not needed in Google, there is no Xen, please remove.

File: google-dracut-config.spec [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+install -D -m 644 etc/dracut.conf.d/11-resume.conf \
+    %{buildroot}%{_sysconfdir}/dracut.conf.d/11-resume.conf
+
+%check
+
+%files
+%doc README.md
+%license LICENSE
+%dir %{_sysconfdir}/dracut.conf.d/
+%config %{_sysconfdir}/dracut.conf.d/07-aws-type-switch.conf
```
Comment (rjschwei) [inline google-dracut-config.spec]: Drop this

File: google-dracut-config.spec [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+%description
+
+%prep
+%autosetup -p1
+
+%build
+
+%install
+install -D -m 644 etc/dracut.conf.d/07-aws-type-switch.conf \
+    %{buildroot}%{_sysconfdir}/dracut.conf.d/07-aws-type-switch.conf
```
Comment (rjschwei) [inline google-dracut-config.spec]: Drop this, only applies to AWS

**Review by rjschwei — CHANGES_REQUESTED** [review]

**Review by rjschwei — APPROVED** [review]

**Comment by jgleissner** [comment]
Hi, sorry I'm a bit late to this, but IMO packages shouldn't install into `/etc` unless absolutely necessary. Dracut supports `/usr/lib/dracut/dracut.conf.d` so that's where the files should go.

**Comment by smarlowucf** [comment]
> Hi, sorry I'm a bit late to this, but IMO packages shouldn't install into `/etc` unless absolutely necessary. Dracut supports `/usr/lib/dracut/dracut.conf.d` so that's where the files should go.

Okay, I will submit PRs with the changes.

**Pull request merged**